database/seedReviews.js

```diff
@@ -0,0 +1,32 @@
+const mysql = require('mysql');
+const faker = require('faker');
+
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'root',
+  password: 'yukimysql',
+  database: 'Reviews_db',
+});
+
+
+const insertReview = function insertReview(homeID, userID) {
+  const reviewDate = faker.date.between('2015-01-01', '2018-10-26');
+  const stars = Math.floor(Math.random() * 6);
+  const flags = 0;
+  const body = faker.lorem.paragraph();
+  const params = [homeID, userID, reviewDate, stars, flags, body];
+  connection.query('INSERT INTO Reviews_tbl (Home_id, User_id, Review_date, Stars, Flags, Body) VALUES (?, ?, ?, ?, ?, ?)', params);
+};
+
+
+const seedReviews = function seedReviews(n) {
+  connection.connect();
+  for (let i = 1; i <= n; i += 1) {
```

Comment:
This seems to seed 1 review per home and each home will have been reviewed by a different user. Might want to have a many to many relationship between homes and users. (otherwise, you can just use 1 table.) Also we need 100 homes.

Reply:
Good catch! I modified the code to seed a random number of reviews (between 1 and 10) per home by a random user with username between 1 and 50. I also updated the script to seed it for 100 homes.